pkg/workloadmeta/README.md

```diff
@@ -0,0 +1,32 @@
+# Workloadmeta Store
+
+This package is responsible for gathering information about workloads and disseminating that to other components.
+
+## Entities
+
+An _Entity_ represents a single unit of work being done by a piece of software, like a process, a container, a kubernetes pod, or a task in any cloud provider, that the agent would like to observe.
+The current workload of a host or cluster is represented by the current set of entities.
+
+Each _Entity_ has a unique _EntityID_, composed of a _Kind_ and an ID.
+Supported kinds include container, pod, and task.
```

Comment:
I wanted to avoid trying to enumerate everything that's supported, since inevitably such a list falls out of date.  I'll update the text to say "Examples of kinds include .." so that it's a bit clearer that the list is illustrative and not exhaustive.